Packs/CybleEvents/Classifiers/classifier-Cyble_Intel_Alert_Mapper.json

```diff
@@ -15,11 +15,11 @@
                     },
                     "simple": ""
                 },
-                "Cyble Events ID": {
+                "Event ID": {
                     "complex": {
                         "accessor": "",
                         "filters": [],
-                        "root": "cybleeventsid",
+                        "root": "eventid",
```

Comment:
@cyble-dev Why did you change the root? When mapping incident fields, you should map the relevant incident field (like "Event ID") to the relevant key in the incident json. So for example if the original json key is named "cybleeventid" for example, the root should contain this value. Same for all of the other keys.